{"uuid": "06f02b14-cf6e-484c-8830-4ec3710d8b73", "children": ["bf63b8ab-e87f-4f6c-825a-ccefe7d83772"], "befores": [{"name": "get_instance", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable\n", "trace": "  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\学习\\todd_study\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\HogwartsSDET17_todd\\testing\\test_Calculator.py\", line 22, in get_instance\n    calc: Calculator = Calculator()\n"}, "start": 1612410765765, "stop": 1612410765765}], "start": 1612410765765, "stop": 1612410765862}